pandas/core/indexes/datetimes.py

```diff
@@ -2241,21 +2241,56 @@ def date_range(start=None, end=None, periods=None, freq='D', tz=None,
     -------
     rng : DatetimeIndex
 
+    Examples
+    --------
+
+    Given the ``start`` and ``end`` mandatory parameters,
+    with default `freq='D'` (Days)
```

Comment:
and for code double backticks. Sorry the documentation is not clear on that, we need to fix it.

There are some other cases in the examples that need to be changed.

Reply:
No Issues with that, I will fix this.